samples/dnn/classification.cpp

```diff
@@ -141,28 +148,52 @@ int main(int argc, char** argv)
         net.setInput(blob);
         //! [Set input blob]
         //! [Make forward pass]
-        Mat prob = net.forward();
-        //! [Make forward pass]
-
-        //! [Get a class with a highest score]
-        Point classIdPoint;
+        double t_sum = 0.0;
+        double t;
+        int classId;
         double confidence;
-        minMaxLoc(prob.reshape(1, 1), 0, &confidence, 0, &classIdPoint);
-        int classId = classIdPoint.x;
-        //! [Get a class with a highest score]
-
-        // Put efficiency information.
-        std::vector<double> layersTimes;
-        double freq = getTickFrequency() / 1000;
-        double t = net.getPerfProfile(layersTimes) / freq;
+        Mat prob = net.forward();
+        for(int i = 0; i < 200; i++) {
+            prob = net.forward();
+
+            //! [Make forward pass]
+
+            //! [Get a class with a highest score]
+            Point classIdPoint;
+            minMaxLoc(prob.reshape(1, 1), 0, &confidence, 0, &classIdPoint);
+            classId = classIdPoint.x;
+            //! [Get a class with a highest score]
+
+            // Put efficiency information.
+            std::vector<double> layersTimes;
+            double freq = getTickFrequency() / 1000;
+            t = net.getPerfProfile(layersTimes) / freq;
+            t_sum += t;
```

Comment:
It is better to use external `TickMeter` instead of `.getPerfProfile()`

Reply:
Fixed.